Improve contributing guidelines (#199)
buildkite/emojis: README.md

> Custom emojis supported by [Buildkite](https://buildkite.com/) which you can use in your build pipelines and terminal output.

Reviewer: ```suggestion
Custom emoji supported by [Buildkite](https://buildkite.com/) which you can use in your build pipelines and terminal output.
```

> ## Contributing new emojis

Reviewer: ```suggestion
## Contributing new emoji
```

If you're using a tool that's not featured here, you can contribute it by following these steps:

1. Prepare a `64x64` PNG image following the [image guidelines](#image-guidelines) below
1. Fork this repo
1. Add the image to the `img-buildkite-64` directory
1. Add it to the top of the `img-buildkite-64.json` file
1. Add it to the top of the table in this markdown file
1. Send a pull request

> Ifwe're missing Unicode emojis, follow the instructions in [docs/updating-unicode.md](docs/updating-unicode.md)

Reviewer: ```suggestion
If we're missing Unicode emoji, follow the instructions in [docs/updating-unicode.md](docs/updating-unicode.md)
```

## Image guidelines

> Buildkite emojis will be shown on both light or dark backgrounds, and at a small size. Try to follow the guidelines below to make sure your emoji looks the best it can ✨

Reviewer: ```suggestion
Buildkite emoji will be shown on both light or dark backgrounds, and at a small size. Try to follow the guidelines below to make sure your emoji looks the best it can ✨
```
